### DISCUSSION TITLE
Github for Picostrap?

### DISCUSSION THREAD

**Discussion by e-orlov** [holger1411/understrap#1558] [General]
Do you already have a Github for Picostrap? I have some issues/questions about it and don't know where to address them. Your email @livecanvas.com seems to be not the right place for such inquiries.

**Comment by bacoords** [comment]
Picostrap is owned and operated by LiveCanvas, not Understrap. While originally they might have forked the Understrap open source project, Picostrap itself is not affiliated with this project.
